src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -226,6 +226,7 @@ private function getTreeContent(
                         'exclude-shadows' => !$mapping->shouldHydrateShadow(),
                         'fields' => implode(',', $mapping->getProperties()),
                         'tree' => true,
+                        'webspace-nodes' => $webspaceNodes,
```

Comment:
this change was needed to ensure that this will also be used in the redirected action.